### PR TITLE
draft tasklist browser notification

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Relates to https://github.com/camunda/product-hub/issues/2061.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/tasklist/userguide/img/tasklist-notifications-toggle.png
Comment (douglasbouttell-camunda) [inline docs/components/tasklist/userguide/img/tasklist-notifications-toggle.png:1]: This control does not exist in 8.5

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 1]
```diff
@@ -56,6 +56,20 @@ Select the **Assigned to me** list to see the tasks that are assigned to you. Se
 
 ![tasklist-claimed-by-me-list](img/tasklist-claimed-by-me-list_light.png)
 
+Tasklist users can now receive a browser notification when new tasks are assigned to them:
+
+![Tasklist notifications banner](./img/tasklist-notifications.png)
+
+The **Don't miss new assignments** banner at the top of the page appears when the user either assigns a new task to themselves, or opens a task that is already assigned to them. To turn on browser notifications, click **Turn on notifications**.
+
+You can turn these notifications off and on by clicking the profile icon in the top right corner of the screen. Toggle the **Browser notifications** setting to receive notifications about new task assignments or turn off notifications:
```
Comment (douglasbouttell-camunda) [inline docs/components/tasklist/userguide/using-tasklist.md:65]: That's not implemented in 8.5. For now you will have to go into your browser settings and disable notifications for that site.

For example, in Google Chrome, you click the icon in the left of the address bar and turning off notifications.

**Review by douglasbouttell-camunda — DISMISSED** [review]

File: docs/components/tasklist/userguide/using-tasklist.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/tasklist/userguide/using-tasklist.md]: Adjusted 👍

**Review by christian-konrad — DISMISSED** [review]
Neat, thx!

**Review by christian-konrad — APPROVED** [review]

**Pull request merged**